changelog/unreleased/change-remove-deprecated-config-options

```diff
@@ -0,0 +1,39 @@
+Change: Removal of Deprecated Config Options
+
+BREAKING CHANGE for developers and administrators:
+The following configuration options have been removed to simplify the configuration process for administrators and improve the overall manageability of the project,
+as they were either not used anymore or only relevant for CI testing:
+
+- `options.ocm.openRemotely`
+- `options.openLinksWithDefaultApp`
+- `options.upload.xhr.timeout`
+- `option.routing`
```

Comment:
```suggestion
- `options.routing`
```

Reply:
since you explicitly show all `options.sidebar` items in the list a few lines below, you could also mention all existing `options.routing` items here. :-)